src/libs/ajax/leonardo/models/disk-models.ts

```diff
@@ -31,37 +32,25 @@ export const diskStatuses: { [label: string]: DiskStatus } = {
   error: { leoLabel: 'Error' },
 };
 
-export interface GetDiskItem {
+export interface RawListDiskItem {
   id: number;
   cloudContext: CloudContext;
   zone: string;
   name: string;
-  serviceAccount: string;
-  samResource: number;
   status: LeoDiskStatus;
   auditInfo: AuditInfo;
   size: number; // In GB
   diskType: GoogleDiskType;
   blockSize: number;
   labels: LeoResourceLabels;
-  formattedBy: string | null;
 }
 
-export interface ListDiskItem {
-  id: number;
-  cloudContext: CloudContext;
-  zone: string;
-  name: string;
-  status: LeoDiskStatus;
-  auditInfo: AuditInfo;
-  size: number; // In GB
-  diskType: GoogleDiskType;
-  blockSize: number;
-  labels: LeoResourceLabels;
+export interface RawGetDiskItem extends RawListDiskItem {
```

Comment:
I don't know if I like these inheriting from one another. The fact that List is a subset of Get is a coincidence, and not always the case. I believe for both apps and runtimes they've diverged, the motivation being the trade off between a fast query with as much info as possibly and a more costly join on the backend for certain fields.

Reply:
i'm for changing it later if needed, but for now I think it adds clarity.

Reply:
I think they are related in that the more detailed one just adds a small number of additional fields.  Using extends makes it clear which fields are added.  But if folks feel strongly, I can switch it back to non-extended pair of types.